ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/MigrationVersionInfo.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.tools.migrations.util;
+
+import static io.confluent.ksql.tools.migrations.util.MetadataUtil.EMPTY_ERROR_REASON;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.confluent.ksql.api.client.Row;
+import io.confluent.ksql.tools.migrations.util.MetadataUtil.MigrationState;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Objects;
+
+public final class MigrationVersionInfo {
+
+  private static final String DATE_FORMAT = "yyyy-MM-dd HH:mm:ss.SSS z";
+  private static final String EMPTY_MIGRATION_TIMESTAMP = "N/A";
```

Comment:
I don't feel strongly. I think we can leave it as is. In all likelihood, the only way users will view the contents of the metadata stream and table will be through `migrations info`, rather than inspecting the stream/table directly.